pkg/util/iptables/iptables.go

```diff
@@ -49,6 +51,18 @@ type Interface interface {
 	DeleteRule(table Table, chain Chain, args ...string) error
 	// IsIpv6 returns true if this is managing ipv6 tables
 	IsIpv6() bool
+	// Save calls `iptables-save` for table.
+	Save(table Table) ([]byte, error)
```

Comment:
the code for this exists in the iptables proxy branch, just need to pull over here.